test/example/packages/mef/src/yang/ietf-network-topology.yang

```diff
@@ -0,0 +1,308 @@
+module ietf-network-topology {

```

Comment:
Why is this model being added to this repository? At best it should be downloaded on demand, and failing that manually downloaded before starting the test.